README.md

```diff
@@ -134,8 +134,8 @@ $ git clone https://github.com/YOUR-USERNAME/YOUR-PROBLEM-S-REPOSITORY
 ```
 Next, you need to [configure the remote](https://help.github.com/articles/configuring-a-remote-for-a-fork/) and [synchronize](https://help.github.com/articles/syncing-a-fork/) it.
 
-Make sure you have synced up local master branch and upstream master branch.
-Since this will keep local master branch up-to-date with the upstream repository.
+Make sure you have synced up local main branch and upstream main branch.
```

Comment:
I know that the originals did not have the markings to make this appear as code, but making `main` look like in this comment by using the backtick can be helpful in suggesting a literal, rather than just an idea. (It is the "main" branch, but also literally the `main` branch.)

Reply:
No problem. Updated main to `main` in my latest commit.